package.json

```diff
@@ -73,6 +73,7 @@
     "@typescript-eslint/eslint-plugin": "^6.10.0",
     "@typescript-eslint/parser": "^6.10.0",
     "antd-theme-generator": "1.2.2",
+    "browser-sync": "^3.0.0",
```

Comment:
Can we remove this?

Reply:
This dependencies is added when we use ng update @angular/core @angular/cli

Reply:
This dependencies was added by the schematics update for @angular/ssr package

Reply:
yes this package was added during the migration. but it seems there is no effect to remove it, i will push the package json file.

Saw you partially approved the PR, do i have to change something more ?

Reply:
Yes, let's remove this dependency if we can.  You can push a change again.

Reply:
done :)